webdriver/tests/bidi/browsing_context/context_destroyed/context_destroyed.py

```diff
@@ -203,16 +219,43 @@ async def on_event(_, data):
         url=test_page_nested_frames, context=new_tab["context"], wait="complete"
     )
 
+    contexts = await bidi_session.browsing_context.get_tree(root=new_tab["context"])
+    top_context = contexts[0]
+    top_iframe = top_context["children"][0]
+    nested_iframe = top_iframe["children"][0]
+
     # Destroy top context
     await bidi_session.browsing_context.close(context=new_tab["context"])
 
-    assert len(events) == 1
+    #
+    assert len(events) == 3
+    # Assert first the most nested iframe was destroyed.
     assert_browsing_context(
         events[0],
-        new_tab["context"],
+        nested_iframe["context"],
+        children=None,
+        url=test_page,
+        parent=top_iframe["context"],
+    )
+
+    # Assert the top-level iframe was destroyed.
```

Comment:
Maybe we can call it `parent` in both tests? We have only three levels which means there is only one parent iframe. Using `top-level` is a bit confusing for browsing contexts because that's usually not a frame.

```suggestion
    # Assert the parent iframe was destroyed.
```